src/components/NcModal/NcModal.vue

```diff
@@ -612,7 +612,6 @@ export default {
 			case 37: // left arrow
 				this.previous(e)
 				break
-			case 13: // enter key
```

Comment:
So pressing enter when nothing is focused, will not go to the next slide anymore ?

Maybe we can add a listener for enter and stop the propagation ? Something like:

```vue
<NcButton v-show="hasPrevious"
	...
	@click="previous"
	@keydown.enter.stop="previous">
	...
</NcButton>
```

Reply:
🤔 the modal is focus trapped so some element will always be focused on open

After it's opened though the user could click on some empty space in the modal which gives focus to `body`, but having to click empty space so that `Enter` goes to the next slide would be strange UX

Reply:
We could also debounce this.
But stopping the propagation should be the solution. To avoid firing further events listeners.

Reply:
Using debounce here to allow for the specific case of clicking empty space to put focus on `body` first before navigating seems unintuitive

Right arrow navigation seems more consistent and predictable